lib/util/asset_token_ext.dart

```diff
@@ -69,6 +69,20 @@ extension AssetTokenExtension on AssetToken {
     }
   }
 
+  String get secondaryMarketName {
+    final url = secondaryMarketURL;
+    if (url.contains(OPENSEA_ASSET_PREFIX)) {
+      return 'OpenSea';
+    } else if (url.contains(FXHASH_IDENTIFIER)) {
+      return 'FXHash';
+    } else if (url.contains(TEIA_ART_ASSET_PREFIX)) {
+      return 'Teia Art';
+    } else if (url.contains(objktAssetPrefix)) {
+      return 'Objkt';
+    }
+    return '';
```

Comment:
it should be default name?

Reply:
No, we only support some pages. If it's not in it, that mean there no url at all